sdk/samples/iot/provisioning/README.md

```diff
@@ -11,50 +11,129 @@ urlFragment: iot-provisioning-samples
 
-Key concepts are explained in detail [here][SDK_README_KEY_CONCEPTS].
+The samples' instructions include specifics for both Windows and Linux based systems.  For Linux, the examples are tailored to Debian/Ubuntu environments.  The samples are also designed to work on MacOS systems, but the instructions do not yet include specific command line examples for this environment.
 
-## Samples for Azure IoT Provisioning Client APIs
+## Key Concepts
 
-This document describes how to use samples and what is done in each sample.
+Further background on the Azure IoT Client library and key concepts are explained in detail in the [Azure IoT Client README](https://github.com/Azure/azure-sdk-for-c/tree/master/sdk/docs/iot#azure-iot-clients).
 
-## Getting Started
+## Prerequisites
 
-Getting started explained in detail [here][SDK_README_GETTING_STARTED].
+* Have an [Azure account](https://azure.microsoft.com/en-us/) created.
+* Have an [Azure IoT Hub](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-create-through-portal) created.
+* Have an [Azure IoT Hub Device Provisioning Service (DPS)](https://docs.microsoft.com/en-us/azure/iot-dps/quick-setup-auto-provision) created.
+* For Windows systems, have [Microsoft Visual Studio](https://visualstudio.microsoft.com/downloads/) installed.
+* Have [git](https://git-scm.com/download) installed.
+* Have [OpenSSL](https://www.openssl.org/source/) installed:
 
-## Examples
+  For Linux based systems (Debian/Ubuntu), we recommend:
+  ```bash
+  sudo apt-get install libssl-dev
+  ```
+  For non-Linux based systems, download the [OpenSSL 1.1.1 LTS](https://www.openssl.org/source/openssl-1.1.1g.tar.gz) command line utility and follow the downloaded INSTALL document.
+* Have the latest version of [CMake](https://cmake.org/download) installed.  For OS specific installation instructions, follow the downloaded README document.
+* Have VCPKG and [Eclipse Paho MQTT C client](https://www.eclipse.org/paho/) installed.  Use the directions [here](https://github.com/Azure/azure-sdk-for-c#development-environment) to download VCPKG and install Paho MQTT.
 
-   Following section document various examples.
+## Getting Started
 
-1. [IoT Provisioning using Paho][samples_paho]: Register a device using the IoT Provisioning client and the [Eclipse Paho MQTT C client][Eclipse_Paho].
+### Paho IoT Provisioning (Certificates)
+This [sample](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/samples/iot/provisioning/src/paho_iot_provisioning_example.c) uses x509 authentication to connect to Azure IoT Hub Device Provisioning Service (DPS).  To easily run this sample, we have provided a script to generate a self-signed device certification used for device authentication. This script is intended **for sample use only and not to be used in production code**.
```

Comment:
nit: should this be a relative link?

Reply:
I can totally do that if you prefer.  I think I put in the full url because I could test the link before committing.